.gitignore

```
@@ -0,0 +1 @@
+**.rock
```

suggestion(non-blocking): the git ignore can use those too:
```suggestion
**.rock
__pycache__
.tox
venv
```